addons/stock/models/stock_inventory.py

```diff
@@ -245,7 +247,10 @@ def _get_quantities(self):
         if self.prefill_counted_quantity == 'zero':
             domain.append(('product_id.active', '=', True))
         if self.is_conflict_inventory:
-            domain.append(('quantity', '<', '0'))
+            if self.lot_ids:
+                domain = expression.AND([domain, [('lot_id', 'in', self.lot_ids.ids)]])
```

Comment:
I think you could put it out of is_conflict_inventory. Could be useful in other situation

Reply:
Ah I remember why I did this, the problem is we need to distinguish between conflict inventories that are due to duplicate SNs (note: we can have a duplicate SN with -1 quantity) vs inventories that are for negative quantities. I could make it so duplicate SN inventories are `is_conflict_inventory = False`, but that doesn't seem right to me.

Reply:
maybe we could do?
if self.lot_ids:
    ...
if self.is_conflict_inventory and not self.lot_ids:
    ....

Reply:
Ah you got me there. Changed it!